loops.js

```diff
@@ -5,8 +5,10 @@
 // Hoera!
 // Hoera!
 // ==========================================
-
-
+const cheers = 'Hoera!';
```

Comment:
Leuk dat je een variabel hiervoor maakt :) maar het is een beetje onnodig kon ook gewoon "Hoera!" loggen xD